### PR TITLE
Visualization update

### PR DESCRIPTION
**Pull request by caballeto**
This PR contains a set of commits that implement the following:
 - merger of several satellite images, handling overlapping regions with different strategies, including `max`, `average`, `last` and `first`
 - generation of tile map service using `gdal2tiles.py` from `gdal` library
 - generating interactive html map with leaflet.js
 - generating interactive virtual globe 
 - generating map overlay during decoding process (updated `main.py`)
 - corresponding tests with sample files
 - documentation with usage examples; 2 decoded images added as sample files, along with TMS
 - fixed issue of relative path running; now created files with interfaces (and tests) could be run from any directory, not only from within `directdemod` directory

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -260,15 +264,30 @@ def usage(err = ""):
                     if '--tle' in [i[0] for i in optlist]:
                         tleFileName = [i[1] for i in optlist if i[0] == '--tle'][0]
 
-                    if not satName is None and not timeRec is None:
```
Comment (7andahalf) [inline main.py]: Why was this deleted? Please keep it at least as a comment. If we decide to remove it, we will remove it properly.

**Review by 7andahalf — CHANGES_REQUESTED** [review]
Thanks!

File: main.py (hunk 1 above)
Comment (caballeto) [reply main.py]: It is not deleted, it seems only to show as it is. I just have changed places with `not` part, because usually you check for None is: if satName is not None. So technically it was left as it is, I've changed only the visualization part.

Comment (caballeto) [reply main.py]: Github just duplicated the line below: `if satName is not None and not timeRec is None:`

Comment (7andahalf) [reply main.py]: I didn't mean just that one line. I meant the visualization part too. It uses some part of the code elsewhere, and only this uses it. So, if we decide to get rid of this, that also should be deleted. So, just keep the old visualization code part in comment and we decide later. :)

Comment (caballeto) [reply main.py]: Okay.

**Review by 7andahalf — APPROVED** [review]
Great! Thank you :)
